combat-trainer.lic

```diff
@@ -19,7 +19,9 @@ class SetupProcess
     echo("  @priority_defense: #{@priority_defense}") if $debug_mode_ct
     @cycle_armors = settings.cycle_armors
     echo("  @cycle_armors: #{@cycle_armors}") if $debug_mode_ct
-    @last_cycle_time = Time.now - 125
+    @cycle_armors_time = settings.cycle_armors_time || 125
```

Comment:
the || 125 will be removed in a followup, I just wanted to be extra safe for now to not break people.

Reply:
actually it should probably go now? shouldn't be needed